### PR TITLE
chore: merge branch `dev` to `flutter`

### PR DESCRIPTION
**Pull request by EvadeMaster**
chore: merge branch `dev` to `flutter`

> **Note**: <br>
> Originally was (1): 🧹 Chore - merge - `dev` to `flutter` (main)
> Originally was (2): 🔖 build: bump version to v0.0.57

# ⏰ Changelog

### CODE REVIEW

File: .github/workflows/crowdin.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ on:
 
 jobs:
   sync-crowdin:
+    if: github.repository_owner == 'revanced'
```
Comment (Ushie) [inline .github/workflows/crowdin.yml:16]: Why is this necessary? 

It is expected for workflows to fail if you don't set the required secrets, and forkers have to explicitly enable workflows in the first place

Comment (EvadeMaster) [reply .github/workflows/crowdin.yml:16]: It would be spamming status failed in `actions` if forker enables workflows for testing, I can revert the changes if you like

**Comment by Ushie** [comment]
Shouldn't the version bump happen in the flutter branch?

**Comment by EvadeMaster** [comment]
> Shouldn't the version bump happen in the flutter branch?

Ah, well, it should, but for now, merging this PR will bump the version in the `flutter` branch. I don't want to `force-branch` the `dev` branch again, and I'm also busy with an upcoming exam.

**Comment by EvadeMaster** [comment]
> **Warning** <br>
> Do not merge this PR until #712 is fixed

**Comment by Ushie** [comment]
> **Warning**
> Commit 6829d3c breaks core functionality

it is immediately visible that functionality is broken, please investigate before merging.
